tests/helper_functions.py

```diff
@@ -317,12 +318,18 @@ def create_mock_response(status_code, text):
     return response
 
 
-def _compare_conda_env_requirements(env_path, req_path):
-    assert os.path.exists(req_path)
+def _read_yml(path):
+    with open(path, "r") as f:
+        return yaml.safe_load(f)
+
 
-    with open(env_path, "r") as f:
-        custom_env_parsed = yaml.safe_load(f)
-    with open(req_path, "r") as f:
-        requirements = f.read().split("\n")
+def _read_lines(path):
+    with open(path, "r") as f:
+        return f.read().split("\n")
```

Comment:
Nit: how about `f.readlines()`?

Reply:
I first tried `f.readlines()`, and then switched to `f.read().split("\n")` to drop line breaks, and then found `f.read().splitlines()` today:

```python
with open("a.txt") as f:
    print(f.read())
    f.seek(0)
    print(f.readlines())
    f.seek(0)
    print(f.read().split("\n"))
    f.seek(0)
    print(f.read().splitlines())
```

Result:

```
with open("a.txt") as f:
    print(f.read())
    f.seek(0)
    print(f.readlines())
    f.seek(0)
    print(f.read().split("\n"))
    f.seek(0)
    print(f.read().splitlines())
```